apps/cowswap-frontend/src/modules/swap/services/swapFlow/index.ts

```diff
@@ -26,32 +25,24 @@ export async function swapFlow(
     return
   }
 
-  if (input.permitInfo && !input.hasEnoughAllowance) {
-    // If token is permittable and there's not enough allowance, get th permit hook
+  logTradeFlow('SWAP FLOW', 'STEP 2: send transaction')
+  tradeFlowAnalytics.trade(input.swapFlowAnalyticsContext)
+  input.swapConfirmManager.sendTransaction(input.context.trade)
```

Comment:
Oh, nice!
I noticed that before we was opening the pending modal after permit signing